contrib_bots/lib/hangman.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import requests
+from six.moves import range
+
+
+class HangmanHandler(object):
+    def usage(self):
+        return '''
+    This bot allows users to play Hangman in streams that the Hangman
+    bot is subscribed to using the following simple commands:
+    - @hangman new: This command starts a new Hangman game.
+    - @hangman guess <letter>: This command lets the user
+    guess a letter for the current Hangman game.
+    - @hangman stop: This command stops the current Hangman game.
+    - @hangman help: This command gives users information
+    about the Hangman bot.
+    '''
+
+    def triage_message(self, message, client):
+        # Return True if we want to (possibly) respond to this message
+        original_content = message['content']
+        is_hangman_command = original_content.lower().startswith('@hangman ')
+
+        return is_hangman_command
+
+    def handle_message(self, message, client, state_handler):
+        original_content = message['content'].lower()
+        original_content_split = original_content.split(" ")
+        if original_content_split[1] == "new":
+            self = get_new_word(self)
```

Comment:
ahh I see what you did here - it definitely should be part of the class not separate function. Please move the function as a helper method inside the bot's class.